ambry-clustermap/src/main/java/com/github/ambry/clustermap/PropertyStoreToDataNodeConfigAdapter.java

```diff
@@ -105,6 +106,25 @@ public List<String> getAllDataNodeNames() {
     return names == null ? Collections.emptyList() : names;
   }
 
+  /**
+   * Get path to ZNode
+   * @param instanceName the name of instance
+   * @return path to ZNode
+   */
+  private String getPath(String instanceName) {
+    return CONFIG_PATH + SEPARATOR + instanceName;
```

Comment:
does not matter here but u may consider string.join if the path grows

Reply:
Yeah, true. Left it for now since path is small.